src/streams/StreamScreen.js

```diff
@@ -59,29 +60,31 @@ class StreamScreen extends PureComponent<Props> {
     const { styles } = this.context;
 
     return (
-      <Screen title="Stream" padding>
-        <StreamCard stream={stream} subscription={subscription} />
-        <OptionDivider />
+      <Screen title="Stream">
+        <View style={styles.padding}>
+          <StreamCard stream={stream} subscription={subscription} />
+        </View>
         <OptionRow
           label="Pinned"
           defaultValue={subscription.pin_to_top}
           onValueChange={this.handleTogglePinStream}
         />
-        <OptionDivider />
         <OptionRow
           label="Muted"
           defaultValue={stream.in_home_view === false}
           onValueChange={this.handleToggleMuteStream}
         />
-        <OptionDivider />
         <OptionRow
           label="Notifications"
           defaultValue={subscription.push_notifications}
           onValueChange={this.toggleStreamPushNotification}
           customStyle={this.context.styles.backgroundColor}
         />
-        <ZulipButton style={styles.marginTop} text="Topics" onPress={this.handleTopics} />
-        <ZulipButton style={styles.marginTop} text="Edit" onPress={this.handleEdit} />
+        <OptionDivider />
+        <View style={styles.screenPadding}>
```

Comment:
Why do you need this? Adding padding here will misalign the buttons from both sides, as they already have padding.

Reply:
But the reason I removed padding from `Screen` in the first place is so that the `OptionDivider` can span the full width of the screen (without having something weird like negative margins). If it did not it would look a bit odd.